core/src/main/java/org/polypheny/db/config/RuntimeConfig.java

```diff
@@ -188,6 +189,11 @@
             BackgroundTask.TaskSchedulingType.EVERY_THIRTY_SECONDS,
             ConfigType.ENUM ),
 
+    FRESHNESSLOOP( "materializedView/freshnessLoopRate",
```

Comment:
Adjust name and include materialized view

Reply:
Renamed to MATERIALIZED_VIEW_LOOP